std/event/channel.zig

```diff
@@ -305,7 +307,8 @@ pub fn Channel(comptime T: type) type {
 
 test "std.event.Channel" {
     // https://github.com/ziglang/zig/issues/1908
-    if (builtin.single_threaded) return error.SkipZigTest;
+    // https://github.com/ziglang/zig/issues/3251
+    if (builtin.single_threaded or freebsd.is_the_target) return error.SkipZigTest;
```

Comment:
This is a minor detail, but would you mind decoupling these? Don't modify the existing comment / if statement. Just add another comment / if statement above or below.

Reply:
Could you give me an example? I'm not able to picture what you want. :crying_cat_face: